api/src/main/java/de/adorsys/keymanagement/api/keystore/KeyStoreOper.java

```diff
@@ -8,5 +8,12 @@
 
 public interface KeyStoreOper extends KeyStoreCreator {
 
+    /**
+     * Adds key entry to keystore.
+     * @param ks keystore where to add key
+     * @param entry key entry which will be added to keystore
```

Comment:
`Key that is going to be added to KeyStore. Note that ProvidedKeyTemplate is the wrapper for KeyStore entries that provides additional information like key name`

Reply:
done